classes/class-wc-connect-api-client.php

```diff
@@ -295,6 +295,32 @@ public function auth_test() {
 			return $this->request( 'GET', '/connection/test' );
 		}
 
+		/** Heartbet test.
```

Comment:
I know this already got merged but can we fix these misspellings @budzanowski

Reply:
@laurendavissmith done.